d3a_api_client/aggregator.py

```diff
@@ -47,6 +49,9 @@ def __init__(self, simulation_id, domain_name, aggregator_name,
         self.aggregator_uuid = None
         self._client_command_buffer = ClientCommandBuffer()
         self._connect_to_simulation()
+        self.latest_grid_tree = {}
+        self.latest_grid_tree_flat = {}
+        self.area_name_uuid_mapping = {}
```

Comment:
Maybe after adding `self.area_name_uuid_mapping` we can get rid of the `self.device_uuid_list` in order to not have separate  objects keeping track of the devices

Reply:
`self.area_name_uuid_mapping` is a `Dict(str, list)` where the uuids are in the values. For the check which devices the aggregator controls an "additional" list is OK IMO, so proposing to keep it as is, except others vote differently.

Reply:
I think we should keep it as it is unless you make the `area_name_uuid_mapping` as `Dict(str, str)` (The case where a name can exist only once in the grid